specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2024-11-01-preview/TuningOptions.json

```diff
@@ -0,0 +1,456 @@
+          },
+          "x-ms-identifiers": [
+            "dimensionName"
+          ],
+          "readOnly": true,
+          "description": "The estimated impact of this recommended action"
+        },
+        "details": {
+          "$ref": "#/definitions/IndexRecommendationDetails",
+          "description": "Stores recommendation details for the recommended action."
+        }
+      },
+      "description": "Index recommendation properties."
+    },
+    "IndexRecommendationResource": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The type of resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "kind": {
+          "description": "Kind of index recommendation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
+          "type": "string",
+          "description": "The id of index recommendation.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of index recommendation.",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/IndexRecommendationResourceProperties",
+          "x-ms-client-flatten": true,
+          "description": "Properties of IndexRecommendationResource."
+        }
+      },
+      "description": "Index recommendation properties."
+    },
+    "IndexRecommendationListResult": {
+      "description": "A list of available index recommendations.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of available index recommendations.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IndexRecommendationResource"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "nextLink": {
+          "description": "URL client should use to fetch the next page (per server side paging).\r\nIt's null for now, added for future use.",
+          "type": "string"
+        }
+      }
+    },
+    "TuningOptionsResource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource id."
+        },
+        "type": {
+          "type": "string",
+          "description": "Tuning options type."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the tuning option."
+        },
+        "properties": {
+          "$ref": "#/definitions/TuningOptionsResourceProperties",
+          "x-ms-client-flatten": true,
+          "description": "Properties of Tuning Options resource."
+        }
+      },
+      "description": "Stores property that features impact on some metric if this recommended action is applied."
+    },
+    "TuningOptionsResourceProperties": {
+      "type": "object",
```

Comment:
>"type": "object" [](http://example.com/codeflow?start=6&length=16)

Empty object/additionalProperties pattern is not allowed. Please schematize this object.

Reply:
Will remove, we do not have props defined for the object and as of now not returning data.